### DISCUSSION TITLE
DeluxeSellwands support

### DISCUSSION THREAD

**Discussion by FourNineRugz** [BG-Software-LLC/WildChests#84] [Denied Feature Requests] [answered]
https://www.spigotmc.org/resources/1-8-1-18-deluxesellwands-need-a-sellwands-plugin-look-no-further.74999/

When using a sellwand on a storage unit. It doesnt detect the number of items in the chest, rather, just the stack that's readily apparent. If possible, make it so sellwand is capable of selling the entire chest no matter how much is in the container.

**Comment by OmerBenGera** [comment, marked as answer]
Ask them to add support
